range-counter.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const sensorData = require('./twelvebit-sensor-data');
+
+function sort(listOfNum) {
+  return listOfNum.sort((a, b) => a - b);
+}
+
+function resetCounterMap(counter, rangeListdt) {
+  counter = 1;
+  rangeListdt.length = 0;
+}
+
+function isInRange(sortedNum, i) {
+  return sortedNum[i + 1] - sortedNum[i] === 1 ? true :
```

Comment:
Accessing index+1 element is an undefined behavior for the last element. Please confirm that this will not pose any problem later if the JS behavior changes.
In other words design should make sure that access to out of bounds elements is not performed. This will ensure consistency.